ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/macro/MacroRegistryImpl.java

```diff
@@ -39,9 +38,7 @@ public MacroRegistryImpl() {
     public void register(Set<Macro> macros) {
         for (Macro macro : macros) {
             final String name = macro.getName();
-            if (this.macros.containsKey(name)) {
-                Log.warn(MacroRegistryImpl.class, "Command macro '" + name + "' is already registered.");
```

Comment:
Exactly! Strongly agree with your opinion.
That's why I urge to log only really helpful messages to the DevConsole.
Messages that appear only when some problem is occurred but not every time.
Messages that may really help developer to solve his problem.
And logging in this registry exactly that case.